runtime/api-channel.js

```diff
@@ -262,6 +262,16 @@ class PECOuterPort extends APIPort {
     this.registerHandler("ArcCreateView", {callback: this.Direct, arc: this.LocalMapped, viewType: this.ByLiteral(Type), name: this.Direct});
     this.registerInitializer("CreateViewCallback", {callback: this.Direct, viewType: this.ByLiteral(Type), name: this.Direct, id: this.Direct});
 
+    this.registerHandler("ArcCreateSlot", {
```

Comment:
Probably best to keep these as one-liners like the rest.

Reply:
done